src/encoded/static/components/typeutils.js

```diff
@@ -373,6 +378,26 @@ const experimentTableColumns = {
         title: 'Assay',
     },
 
+    'replicates.library.biosample': {
+        title: 'Biosamples',
+        display: (item) => {
+            if (item.replicates.length > 0) {
+                const biosamples = item.replicates.reduce((accBiosamples, replicate) => accBiosamples.concat(replicate.library.biosample), []);
+                if (biosamples.length > 0) {
+                    return biosamples.map((biosample, index) => (
+                        <span key={biosample['@id']}>
+                            {index > 0 ? <>, </> : null}
+                            <a href={biosample['@id']}>{biosample.accession}</a>
+                        </span>
+                    ));
+                }
+                return null;
+            }
+            return null;
+        },
+        hide: (list, columns, meta) => !(meta && meta.showBiosamples),
+    },
```

Comment:
The biosamples aren't being concatenated quite right here.
Look at this example: https://encd-5675-biosample-hierarchy-flat-qa1-fytanaka.demo.encodedcc.org/biosamples/ENCBS056AAA/
Some of the `, `s between accessions are mixed up.

![Screen Shot 2021-02-10 at 9 12 43 PM](https://user-images.githubusercontent.com/2887343/107604580-6caea600-6be5-11eb-887f-7552ff99141f.png)

Reply:
Furthermore, do we expect there to be repeated biosamples? We don't want to use `.uniq` or `Set` to get a list without repeated biosample links?

Reply:
Ah, I didn't realize you could have repeats here, and it caused a React keys problem I believe. I fixed that, but I found an issue that might cause me to return to doing an initial search request, or embed a new property. I'll find out Thursday.